Jenkinsfile

```diff
@@ -0,0 +1,21 @@
+pipeline {
+    agent label:"docker", docker: "maven:3.3.9-jdk-7"
+
+    environment {
+      MAVEN_OPTS = "-XX:MaxPermSize=256m -Xmx1024m"
+    }
+
+    stages {
+        stage("Build") {
+            sh 'mvn clean install -Dmaven.test.failure.ignore=true'
+        }
+    }
+
+    postBuild {
+        always {
```

Comment:
Or cause a spurious and confusing failure in the post-build section after a compilation failure? I would think you would want to do this only if the main build was a success.